src/cargo/ops/registry.rs

```diff
@@ -344,7 +343,6 @@ pub fn registry_configuration(
     // `registry.default` is handled in command-line parsing.
     let (index, token, process) = match registry {
         Some(registry) => {
-            validate_package_name(registry, "registry name", "")?;
```

Comment:
Was this intentional to be removed? (in isolation I don't know what this is doing and it may no longer be necessary but otherwise it seems out of place for a cleanup-style commit)

Reply:
Yes, I tryed to call it out in the PR:
> The only thing nontrivial was noticing that get_registry_index already calls validate_package_name.

So the next line calls this function.